locale/in/LC_MESSAGES/index.po

```diff
@@ -0,0 +1,2795 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2021 Fortran programming language community
+# This file is distributed under the same license as the fpm package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: fpm \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-10 00:47+0530\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: KRITIKA RAG <kritikarag.kr@gmail.com>\n"
+"Language: in\n"
+"Language-Team: Hindi <https://hosted.weblate.org/projects/fortran-lang/fpm/"
+"in/>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Weblate 4.14.2-dev\n"
+"Generated-By: Babel 2.12.1\n"
+
+#: ../../pages/design/index.md:3
+msgid "Design documents"
+msgstr "डिजाइन दस्तावेज"
+
+#: ../../pages/design/index.md:6
+msgid ""
+"This section contains the resources around the design of the Fortran Package "
+"Manager (fpm)."
+msgstr ""
+"इस अनुभाग में फोरट्रान पैकेज प्रबंधक (एफपीएम) के डिज़ाइन से संबंधित संसाधन शामिल हैं।"
+
+#: ../../pages/design/index.md:9
+msgid ""
+"Fortran Package Manager (fpm) is a package manager and build system for "
+"Fortran. Its key goal is to improve the user experience of Fortran "
+"programmers. It does so by making it easier to build your Fortran program or "
+"library, run the executables, tests, and examples, and distribute it as a "
+"dependency to other Fortran projects. Fpm's user interface is modeled after "
+"[Rust's Cargo](https://doc.rust-lang.org/cargo/). Its long term vision is to "
+"nurture and grow the ecosystem of modern Fortran applications and libraries."
+msgstr ""
+"फोरट्रान पैकेज मैनेजर (एफपीएम) फोरट्रान के लिए एक पैकेज प्रबंधक और बिल्ड सिस्टम है। "
+"इसका मुख्य लक्ष्य फोरट्रान प्रोग्रामर के उपयोगकर्ता अनुभव को बेहतर बनाना है। यह आपके "
+"फोरट्रान प्रोग्राम या लाइब्रेरी को बनाने, निष्पादन योग्य, परीक्षण और उदाहरण चलाने और इसे "
+"अन्य फोरट्रान परियोजनाओं के लिए निर्भरता के रूप में वितरित करना आसान बनाकर ऐसा करता है। "
+"एफपीएम का यूजर इंटरफेस [रस्ट कार्गो](https://doc.rust-lang.org/cargo/) पर आधारित है| "
+"इसकी दीर्घकालिक दृष्टि आधुनिक फोरट्रान अनुप्रयोगों और लाइब्रेरीयों के पारिस्थितिकी तंत्र का देखभाल "
+"और विकास करना है। "
```

Comment:
Phonetic conversion of programming terms?

Reply:
@HaoZeke Okay I did said that I used translation tools because it's really hard to type hindi letters using an english keyboard, but I'm really sorry to say that your claim that the whole document was auto-translated in one go is simply incorrect. The whole translation took me 13 days to finish, I literally had been reviewing each and every word of this translation and had been working on the semantics (I was trying on chagpt, google translate and microsoft translate but hardly any of them were giving a proper translation with proper semantics, I myself had to make changes in the words in each line). For the phonetic conversion of programming terms I had already stated wasn't sure that whether I should write them in English or in Hindi (I've seen both on different sites) so I just left it as it was and had been waiting since days for someone to review it (I got a review from another native community member like 2 weeks ago with a green signal) and now also I'm ready to do as per suggested but again I would apologize the translation that I've made is not just totally auto translated and something that was just done in one shot. But fine if the only solution is to rewrite it(full code or just some sections of it) then I'll do it (again I'm not sure how am I gonna do it simply without not using auto-translation but I'll figure something out), just I don't think I'll able to work on this up until next week because my final semester exams are going on.

Reply:
@kritikarag we appreciate your work! 13 days is a lot of time, thank you for putting so much effort into it. I am sorry if @HaoZeke or my feedback felt critical. All we want is a high quality translation that makes sense to maintain. Let's keep this PR open and let's simply break it up into smaller chunks and several PRs that several native speakers can review and if they all agree that the translation is good and natural, then we merge it one by one. You can start with my suggestion below, that is just one paragraph and that can be done, and then once we get into the workflow it will go fast. Take your time, there is no rush. If you have any questions or doubts, or want to discuss it more, I am always available.

Reply:
@certik No that's totally fine infact I'm really grateful for your suggestions and will start working on that paragraph keeping your suggestions in my mind ASAP.